lightning-invoice/src/utils.rs

```diff
@@ -1015,4 +1201,81 @@ mod test {
 		}
 		assert!(chan_ids_to_match.is_empty(), "Unmatched short channel ids: {:?}", chan_ids_to_match);
 	}
+
+    #[test]
+    fn test_create_invoice_with_description_hash() {
```

Comment:
I'd personally place this test with the other tests that doesn't require the `std` config, just so it becomes clearer what's actually loaded under which config. I suggest you place it after the `test_from_channelmanager` test.

Reply:
[d1f4536](https://github.com/lightningdevkit/rust-lightning/pull/1361/commits/d1f4536ddecfde661b99777903a9af14b58064f7)